thirdparty/czmq/CMakeLists.txt

```diff
@@ -16,9 +16,8 @@ assert_var_defined(HOST)
 ep_get_source_dir(SOURCE_DIR)
 ep_get_binary_dir(BINARY_DIR)
 
-#-Wno-error so we can build on glibc 2.24 in spite of readdir_r
-set(CFG_ENV_VAR "CC=\"${CC}\" LDFLAGS=\"${LDFLAGS}\" CFLAGS=\"-Wno-error ${CFLAGS}\" czmq_have_xmlto=no czmq_have_asciidoc=no")
-set(CFG_OPTS "-q --prefix=${BINARY_DIR} --with-gnu-ld --with-libzmq=${ZMQ_DIR} --disable-static --enable-shared --host=${HOST}")
+set(CFG_ENV_VAR "CC=\"${CC}\" CFLAGS=\"${CFLAGS} -I${ZMQ_DIR}/include\" LDFLAGS=\"${LDFLAGS} -L${ZMQ_DIR}/lib\" czmq_have_xmlto=no czmq_have_asciidoc=no")
```

Comment:
I got this change from [the czmq readme](https://github.com/zeromq/czmq/blob/eaac91a272779299d0b9ab8ac62c7dee1c34f567/README.md#building-and-installing):
```
export CFLAGS=-I$HOME/local/include
export LDFLAGS=-L$HOME/local/lib64
export PKG_CONFIG_PATH=$HOME/local/lib64/pkgconfig
./configure
```
Oddly enough it works although I'm really just restating what the --with-libzmq flag is supposed to do.

Anyway, the same fix doesn't work on filemq and zyre and I have no idea why it's not picking up on the libraries correctly.